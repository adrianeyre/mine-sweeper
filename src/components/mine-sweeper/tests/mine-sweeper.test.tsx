import React from 'react';
import { shallow } from 'enzyme';

import MineSweeper from '../mine-sweeper';
import IMineSweeperProps from '../interfaces/mine-sweeper-props';

describe('Mine Sweeper', () => {
	it('Should render correctly', () => {
		const defaultProps: IMineSweeperProps = {};
		const fishy = shallow(<MineSweeper {...defaultProps} />);
		expect(fishy).toMatchSnapshot();
	});
});