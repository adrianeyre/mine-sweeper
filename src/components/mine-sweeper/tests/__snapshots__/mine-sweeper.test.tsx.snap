// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mine Sweeper Should render correctly 1`] = `ShallowWrapper {}`;
